test/distributed/org/apache/cassandra/distributed/impl/Versions.java

```diff
@@ -168,13 +168,15 @@ public Version getLatest(Major major)
 
     public static Versions find()
     {
-        logger.info("Looking for dtest jars in " + new File("build").getAbsolutePath());
+        final String dtestJarDirectory = System.getProperty("org.apache.cassandra.distributed.test.jar_path","build");
+        final File sourceDirectory = new File(dtestJarDirectory);
```

Comment:
style comment (aka feel free to ignore); could simplify this to be `new File(System.getProperty("org.apache.cassandra.distributed.test.jar_path","build"))`